src/foam/util/UIdGenerator.js

```diff
@@ -0,0 +1,118 @@
+/**
```

Comment:
file name should match the 'name' property.

Reply:
addressed. Could I get a re-review? Thanks @jlhughes @KernelDeimos